src/plugins/intel_cpu/src/emitters/jit_snippets_emitters.cpp

```diff
@@ -978,29 +1017,210 @@ void BrgemmEmitter::emit_isa(const std::vector<size_t> &in, const std::vector<si
+    using Vmm = typename dnnl::impl::utils::conditional3<isa == dnnl::impl::cpu::x64::sse41,
+            Xmm, isa == dnnl::impl::cpu::x64::avx2, Ymm, Zmm>::type;
+
+    Vmm src_vmm = Vmm(in[0]);
+    Xmm dst_xmm = Xmm(out[0]);
+    Xmm aux_xmm = Xmm(aux_vec_idxs[0]);
+
+    Reg64 aux_reg = Reg64(aux_gpr_idxs[0]);
+    Reg32 aux_reg_32 = Reg32(aux_reg.getIdx());
+
+    const size_t vlen = dnnl::impl::cpu::x64::cpu_isa_traits<isa>::vlen;
+    const size_t vec_size = vlen / sizeof(float);
+    h->sub(h->rsp, vlen);
+    h->uni_vmovups(h->ptr[h->rsp], src_vmm);
+    h->uni_vpxor(dst_xmm, dst_xmm, dst_xmm);
+    for (size_t i = 0; i < vec_size; i++) {
+        h->mov(aux_reg, h->ptr[h->rsp + i * sizeof(float)]);
+        h->vmovq(aux_xmm, aux_reg);
+        h->uni_vaddps(dst_xmm, dst_xmm, aux_xmm);
+    }
+    h->add(h->rsp, vlen);
+}
+
+ZeroEmitter::ZeroEmitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa, const std::shared_ptr<ov::Node>& n) :
+    jit_emitter(h, isa, n, Precision::FP32, emitter_in_out_map::vec_to_vec) {}
+
+void ZeroEmitter::emit_impl(const std::vector<size_t>& in,
+                                    const std::vector<size_t>& out,
+                                    const std::vector<size_t>& pool,
+                                    const std::vector<size_t>& gpr,
+                                    const ov::intel_cpu::emitter_context *emit_context) const {
+    if (host_isa_ == dnnl::impl::cpu::x64::sse41) {
+        emit_isa<dnnl::impl::cpu::x64::sse41>(in, out);
+    } else if (host_isa_ == dnnl::impl::cpu::x64::avx2) {
+        emit_isa<dnnl::impl::cpu::x64::avx2>(in, out);
+    } else if (host_isa_ == dnnl::impl::cpu::x64::avx512_core) {
+        emit_isa<dnnl::impl::cpu::x64::avx512_core>(in, out);
+    } else {
+        IE_THROW() << "Zero emitter doesn't support " << host_isa_;
+    }
+}
+
+template <dnnl::impl::cpu::x64::cpu_isa_t isa>
+void ZeroEmitter::emit_isa(const std::vector<size_t> &in, const std::vector<size_t> &out) const {
+    using Vmm = typename dnnl::impl::utils::conditional3<isa == dnnl::impl::cpu::x64::sse41,
+            Xmm, isa == dnnl::impl::cpu::x64::avx2, Ymm, Zmm>::type;
+
+    Vmm vmm = Vmm(out[0]);
+    h->uni_vpxor(vmm, vmm, vmm);
+}
+
+FillEmitter::FillEmitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa, const std::shared_ptr<ov::Node>& n) :
+    jit_emitter(h, isa, n, Precision::FP32, emitter_in_out_map::vec_to_vec) {
+    const auto fill = ov::as_type_ptr<ngraph::snippets::op::Fill>(n);
+    if (fill->get_element_type().size() != 4) {
+        IE_THROW() << "Fill emitter supports only 4 Byte element types but gets: " << fill->get_element_type();
+    }
+
+    offset = fill->get_offset();
+    fill_value = fill->get_fill_value();
+    prepare_table();
+}
+
+size_t FillEmitter::aux_gprs_count() const {
+    // + 1 reg for mask on avx512
+    return one_of(host_isa_, dnnl::impl::cpu::x64::avx512_core) ? 2 : 1;
```

Comment:
2 for AVX512 is needed because we should move `tail_mask` to `k_mask` using `aux_gpr_reg` (because of that I left comment about that) and one more for constant. As for mask, I copied this code part from [load_emitter](https://github.com/openvinotoolkit/openvino/blob/67f467b2f3f9a5106023ac4c754736b8e0792477/src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp#L70)